samples/go/photo-dedup/job/hashjob.go

```diff
@@ -33,51 +33,50 @@ func HashPhotosJob(db datas.Database, photoSets []types.Value, outDS datas.Datas
 
 // hashPhotos adds a dhash to each photo in photoSets and delivers them on the returned channel
 func hashPhotos(db datas.Database, photoSets []types.Value) <-chan types.Struct {
-	photosToHashCh := make(chan model.Photo)
-	hashedPhotosCh := make(chan types.Struct)
+	numWorkers := runtime.NumCPU() * 4
```

Comment:
I see both `NumCPU()*4` and `NumCPU() * 4` in this code. I'm surprised `gofmt` doesn't have an opinion on this...